src/modules/CRUD/services/CRUD.js

```diff
@@ -0,0 +1,51 @@
+import Api from '@terralego/core/modules/Api';
+
+export async function fetchCRUDSettings () {
+  return Api.request('crud/settings');
+}
+
+export function getLayerFromCRUD ({ menu = [] }, name) {
```

Comment:
*function* is only for _Class methods_.

In any other case: _Arrow function_ should be preferred.


```suggestion
export const getLayerFromCRUD = ({ menu = [] }, name) => {
```